templates/manifest.json

```diff
@@ -121,22 +121,24 @@
             ]
         },
         {
-            "id": "js-start",
-            "name": "getting_started_node",
-            "label": "Start with JavaScript",
-            "description": "A simple JavaScript example of core Actor and Apify SDK features. It reads and validates user input with schema, computes a result and saves it to storage.",
+            "id": "js-scrape-single-page",
+            "name": "js_scrape_single_page",
+            "label": "Scrape single page in JavaScript",
+            "description": "Scrape single page with provided URL with Axios and extract data from page's HTML with Cheerio.",
             "category": "javascript",
             "technologies": [
-                "nodejs"
+                "nodejs",
+                "cheerio",
+                "axios"
             ],
-            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/js-start.zip?raw=true",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/js-scrape-single-page.zip?raw=true",
             "defaultRunOptions": {
                 "build": "latest",
                 "memoryMbytes": 256,
                 "timeoutSecs": 360
             },
             "showcaseFiles": [
-                "src/main.js"
+                "main.js"
```

Comment:
I think we (me, lukas, martin, some other ppl on Slack) agreed some time ago that all code will always be in `src`

Reply:
@mnmkng I thought it was only if there are more source files, not just the main one. Is this really what you wanted?